bin/pubsub_cli.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Pub/Sub manipulation script.
+#
+# Note: Run this in the base container which has access to Pub/Sub.
+#
+# Usage: ./bin/pubsub_cli.py [SUBCOMMAND]
+
+import click
+from google.cloud import pubsub_v1
+from google.api_core.exceptions import AlreadyExists, NotFound
+
+
+@click.group()
+def pubsub_group():
+    """Local dev environment Pub/Sub emulator manipulation script."""
+
+
+@pubsub_group.command("list_topics")
+@click.argument("project_id")
+@click.pass_context
+def list_topics(ctx, project_id):
+    """List topics for this project."""
+    raise NotImplementedError(
+        "pubsub emulator times out for list_topics as of gcloud cli 463.0.0"
+    )
+
+    print("Listing topics in project %s:" % project_id)
```

Comment:
Nit: If we're using click, we can use `click.echo` which allows us to do ansi color codes, terminal width wrapping, etc. Also, I switched to using f-strings everywhere except for logging. f-strings are kind of quirky, but work pretty nicely and are easier to read most of the time.

```suggestion
    click.echo(f"Listing topics in project {project_id}.")
```